functions/Get-CSVMetadata.ps1

```diff
@@ -0,0 +1,283 @@
+﻿function Get-UserPrompt {

```

Comment:
Is Get-CsvMetadata intended to be a public function or an internal one? If public, please go through Trello first when proposing new commands, as they must get prior approval. If private, please add to SharedFunctions.ps1 (which will soon be moved to a new folder and broken down by command.)

This seems like a pretty cool command overall, can you give details about how it will be used? If we ultimately agree on its use, please rename to Parse-DbaCsvMetadata and change into an advanced command with documentation, a BEGIN (this is where internal functions go), PROCESS and END. You can see this in the [dbatools-templates](https://github.com/sqlcollaborative/dbatools-templates) repo. 

Sorry this wasn't clarified before, I plan to work on the onboarding documentation in the wiki this week.